tests/php/sync/test_class.jetpack-sync-terms.php

```diff
@@ -134,6 +134,21 @@ public function test_delete_object_term_relationships() {
 		$this->assertEquals( $object_terms, $server_object_terms );
 	}
 
+	function test_returns_term_object_by_id() {
+		$term_sync_module = Modules::get_module( 'terms' );
+
+		$event = $this->server_event_storage->get_most_recent_event( 'jetpack_sync_add_term' );
+		$synced_term = $event->args[0];
+
+		$codec = $this->sender->get_codec();
+
+		$retrieved_term = $codec->decode( $codec->encode(
```

Comment:
Could you elaborate on what's the idea behind decoding and encoding the term here?

Reply:
Yes, done!